src/MessageHandler/ActivityPub/Inbox/DeleteHandler.php

```diff
@@ -53,23 +63,17 @@ public function __invoke(DeleteMessage $message): void
 
         $object = $this->entityManager->getRepository($object['type'])->find((int) $object['id']);
 
```

Comment:
add `if ($object)`? or something..

Reply:
above it:
```php
        if (!$object) {
            return;
        }
```

Reply:
No.. after this call .. Who says that `$object = $this->entityManager->getRepository($object['type'])->find((int) $object['id']);` is not returning empty/null data?

Reply:
null checks are covered by the `instanceof` underneath. though I put the result in a different variable to not reassign `$object` to a different datatype